source/openstack/openstack_openpower.rst

```diff
@@ -12,14 +12,42 @@ We use Chef as our configuration management tool of choice and have created an e
 Cookbooks. The compute nodes are running CentOS 7 and the controller node is an x86 virtual machine running CentOS 7.
 All storage for the cluster is powered via a `Ceph`_ cluster backed by a 40Gb network.
 
-We have created our own site specific wrapper `Openstack cookbook`_ that also includes any changes needed to make
-Openstack run on the OpenPOWER platform on CentOS 7. It currently requires some private cookbooks to run however we are
+We have created our own site specific wrapper `OpenStack cookbook`_ that also includes any changes needed to make
+OpenStack run on the OpenPOWER platform on CentOS 7. It currently requires some private cookbooks to run however we are
 working on removing those requirements.
 
 .. _OpenStack Chef: https://docs.openstack.org/openstack-chef/latest/
-.. _Openstack cookbook: https://github.com/osuosl-cookbooks/osl-openstack
+.. _OpenStack cookbook: https://github.com/osuosl-cookbooks/osl-openstack
 .. _Ceph: https://ceph.com/
 
+Current Images
+--------------
+
+We are currently maintaining ppc64le Little endian (LE) and some ppc64 Big Endian (BE) images. Please keep in
+mind that IBM is focusing their efforts primarily on LE so don't expect BE support to last for much longer on
+various distributions.
+
+The list of operating systems we currently support are (systems which support POWER9 are noted with **P9**):
+
+- CentOS 7.6 BE
+- CentOS 7.6 LE (P9)
+- Debian 8.11 BE
+- Debian 8.11 LE
+- Debian 9.9 LE (P9)
+- Debian 10.0 LE (P9)
+- Fedora 29 LE (P9)
+- Fedora 30 LE (P9)
+- OpenSUSE 13.2 BE
+- OpenSUSE 13.2 LE
+- OpenSUSE Leap 15.0 LE (P9)
+- OpenSUSE Leap 42.3 LE
+- Ubuntu 14.04 BE
+- Ubuntu 14.04 LE
+- Ubuntu 16.04 BE
+- Ubuntu 16.04 BE (32bits)
+- Ubuntu 16.04 LE
+- Ubuntu 18.04 LE (P9)
```

Comment:
This list is now replaced with the table you made, can you please remove this?

Reply:
This list goes into more detail on big and little endian availiability. I did not want to clutter the main table with it, but I can add BE/LE into it.

Reply:
Hmm, good point. Why don't you convert this into a table as well then and also mention on the other table about this difference being show here?